tests/load/metrics.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package load
+
```

Comment:
I personally like the metrics split apart from the tracker, but it is also I think an unnecessary change, as long as we share the prometheus registry right? It is clearer that metrics are shared though.

Reply:
Indeed, as the code is right now it should be fine, but that feels like a future hard-to-debug problem if the orchestrator code changes (or other orchestrators are added) given we read the tracker state to adapt the orchestrator.